cli/cppcheckexecutor.h

```diff
@@ -112,7 +112,7 @@ class CppCheckExecutor : public ErrorLogger {
     /**
      * Execute a shell command and read the output from it. Returns true if command terminated successfully.
      */
-    static bool executeCommand(std::string exe, std::vector<std::string> args, const std::string &redirect, std::string *output_);
```

Comment:
I think this should have the same signature as the executeCommand in gui/checkThread. It's used for the same callback. It's called from lib/cppcheck.cpp

Reply:
I have no adjusted this yet.